test/dummy/config/dynamo_db_session_store.yml

```diff
@@ -0,0 +1,70 @@
+# Uncomment and manipulate the key value pairs below
```

Comment:
Should this not be checked in?  Assume its getting generated by the tests

Reply:
I dumped these files in /dummy/ so they could be used with the config loading tests (action_dispatch/session)